devyiitestdrive/protected/controllers/TimeZoneController.php

```diff
@@ -11,15 +11,14 @@ class TimeZoneController extends Controller
 
     function __construct()
     {
-        Yii::import('application.services.*.*');
         $this->timeZoneService = new TimeZoneServiceImpl();
```

Comment:
please don't forget to include the line after overriding the parent method.

```
parent::__construct();
```

ref: https://www.yiiframework.com/doc/api/1.1/CController#__construct-detail

Reply:
Cannot overide the parent constructor it needs one or two argument named $id and $module=null where can I get this?